models/Discord/Commands/DiscordCommandWhoIs.js

```diff
@@ -34,50 +34,50 @@ class DiscordCommandWhoIs extends DiscordCommand {
       var DB_response;
       
       if(userID) {
-        DB_response = getByDiscordID(connection, userID)
+        DB_response = getByDiscordID(connection, userID, message)
       } else {
         DB_response = getByCkey(connection, provided_ckey, message)
       }
         
-      message.reply(DB_response)
       connection.release();
     });
   }
 
 }
 
-function getByDiscordID(connection, user) {
+function getByDiscordID(connection, user, message) {
   connection.query("SELECT * FROM `erro_player` WHERE `discord_id` = ?", [user.id], (error, results, fields) => {
       if (error) {
-        return "Error running select query, try again later.";
+        message.reply( "Error running select query, try again later.");
       }
       if (results.length == 0) {
-        return "No linked BYOND account found for this user.";
+        message.reply("No linked BYOND account found for this user.");
       }
       if (results.length > 1) {
-        return "More than one BYOND account linked to this ID. This shouldn't happen!";
+        message.reply("More than one BYOND account linked to this ID. This shouldn't happen!");
       }
 
       var ckey = results[0].ckey
-      return (user + " belongs to the ckey '" + ckey +"'")
+      message.reply(user + " belongs to the ckey '" + ckey +"'");
     })
 }
 
 function getByCkey(connection, ckey, message) {
   connection.query("SELECT discord_id FROM `erro_player` WHERE `ckey` = ?", [ckey], (error, results, fields) => {
       if (error) {
-        return "Error running select query, try again later.";
+        message.reply("Error running select query, try again later.");
       }
       if (results.length == 0) {
-        return "No user with this Ckey has a linked Discord account!";
+        message.reply("No user with this Ckey has a linked Discord account!");
       }
       if (results.length > 1) {
-        return "More than 1 of this ckey with a Discord ID, this makes no sense at all!";
+        message.reply("More than 1 of this ckey with a Discord ID, this makes no sense at all!");
+
       }
       var guildMember = message.guild.fetchMember(results[0].discord_id);
-      if(!guildMember) return "Can't resolve user from ID"
+      if(!guildMember) message.reply("Can't resolve user from ID");
```

Comment:
make sure you keep a `return` in all the places where you used to have one, or else the code will continue to be executed after this reply.

Reply:
Right, brain fart

Reply:
Actually it was because I can’t tab on my phone .-.